UnitsNet.Tests/CustomCode/AmountOfSubstanceTests.cs

```diff
@@ -59,5 +60,21 @@ public class AmountOfSubstanceTests : AmountOfSubstanceTestsBase
         protected override double NanopoundMolesInOneMole => 0.002204622621848776 * 1e9;
         protected override double PoundMolesInOneMole => 0.002204622621848776;
         protected override double MegamolesInOneMole => 1e-6;
+
+        [Fact]
+        public void NumberOfParticlesInOneMoleEqualsAvogadroConstant()
+        {
+            var oneMole = AmountOfSubstance.FromMoles(1);
+            var numberOfParticles = oneMole.NumberOfParticles();
+            Assert.Equal(AmountOfSubstance.AvogadroConstant, numberOfParticles);
+        }
+
+        [Fact]
+        public void NumberOfParticlesInTwoMolesIsDoubleAvogadroConstant()
```

Comment:
I don't immediately see the value of this test, since the test above it already establishes the value of this constant, but I guess it does confirm that there is a linear relationship between the two.
We can keep it, but fix typo `oneMole` to `twoMoles`.

Reply:
Fixed.